server/src/test/java/com/google/coffeehouse/common/StorageHandlerTest.java

```diff
@@ -261,142 +262,185 @@ public void insertBookWithNullAuthorAndEmptyIsbn() {
-    String expected = "Book ID: bookWithNullIsbn || Author: bell hooks || "
-                        + "No ISBN || Title: all about love: new visions\n";
-    assertEquals(expected, actual);
-  }
-
-  @Test
-  public void getBookQuery_existsWithNullAuthorAndNullIsbn() throws Exception {
-    insertBookWithNullAuthorAndNullIsbn();
-    String actual = StorageHandler.getBookQuery(dbClient, "bookWithNullAuthorAndNullIsbn");
-    String expected = "Book ID: bookWithNullAuthorAndNullIsbn || No author || "
-                        + "No ISBN || Title: anonymous book about love\n";
-    assertEquals(expected, actual);
+  public void insertPerson() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertBuilder("Persons")
+        .set("userId")
+        .to("person")
+        .set("email")
+        .to("person@test.com")
+        .set("nickname")
+        .to("person")
+        .set("pronouns")
+        .to("she/he/they")
+        .build());
+    dbClient.write(mutations);
   }
 
   @Test
-  public void getBookQuery_existsWithEmptyAuthor() throws Exception {
-    insertBookWithEmptyAuthor();
-    String actual = StorageHandler.getBookQuery(dbClient, "bookWithEmptyAuthor");
-    String expected = "Book ID: bookWithEmptyAuthor || No author || "
-                        + "ISBN: 9780060959470 || Title: all about love: new visions\n";
+  public void addPersonClubMembershipMutation() throws Exception {
+    insertPerson();
+    StorageHandler.addPersonClubMembershipMutation(dbClient, "person", "bellhooksbooks");
+    Boolean actual = StorageHandlerHelper.checkPersonInClubQuery(dbClient, "person", "bellhooksbooks");
+    Boolean expected = true;
     assertEquals(expected, actual);
   }
 
   @Test
-  public void getBookQuery_existsWithEmptyIsbn() throws Exception {
-    insertBookWithEmptyIsbn();
-    String actual =StorageHandler.getBookQuery(dbClient, "bookWithEmptyIsbn");
-    String expected = "Book ID: bookWithEmptyIsbn || Author: bell hooks || "
-                        + "No ISBN || Title: all about love: new visions\n";
+  public void deletePersonClubMembership() throws Exception {
+    insertPerson();
+    StorageHandler.addPersonClubMembershipMutation(dbClient, "person", "bellhooksbooks");
+    StorageHandler.deletePersonClubMembershipDml(dbClient, "person", "bellhooksbooks");
+    Boolean actual = StorageHandlerHelper.checkPersonInClubQuery(dbClient, "person", "bellhooksbooks");
+    Boolean expected = false;
     assertEquals(expected, actual);
   }
 
-  @Test
-  public void getBookQuery_existsWithEmptyAuthorAndEmptyIsbn() throws Exception {
-    insertBookWithEmptyAuthorAndEmptyIsbn();
-    String actual = StorageHandler.getBookQuery(dbClient, "bookWithEmptyAuthorAndEmptyIsbn");
-    String expected = "Book ID: bookWithEmptyAuthorAndEmptyIsbn || No author || "
-                        + "No ISBN || Title: anonymous book about love\n";
-    assertEquals(expected, actual);
-  }
+  // @Test
```

Comment:
nit: Don't commit commented-out code. If you think you might need it later, save it locally somewhere.